packages/afp/.github/ISSUE_TEMPLATE/test.md

```
@@ -0,0 +1,14 @@
+---
+name: Test
+about: Adding missing tests or correcting existing tests
+title: "test: "
+labels: test
+---
+
+**Description**
+
+List out the tests that need to be added or changed. Please also include any information as to why this was not covered in the past.
+
+**Requirements**
```

Markdownlint

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)